extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -196,6 +197,13 @@ export class ConsentUI {
           promptUI
         );
       }
+      // Warn of use of <amp-iframe> within a promptUI element.
+      if (!isEsm() && promptElement.querySelector('amp-iframe')) {
```

Comment:
Why is `isEsm` important here?

Reply:
It's merely to exclude the message from optimized builds.